tests/PHPUnit/System/expected/test_OneVisitorTwoVisits_hideColumns___API.getProcessedReport_day.xml

```diff
@@ -51,5 +51,7 @@
 		<entry_bounce_count>1</entry_bounce_count>
 		<entry_nb_actions>8</entry_nb_actions>
 		<exit_nb_visits>2</exit_nb_visits>
+		<nb_uniq_visitors>3</nb_uniq_visitors>
```

Comment:
Here an example where one can see that it fixes report total calculation as mentioned before.